custom/reach/templates/reach/reports/unified_beneficiary.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'reach/dashboard.html' %}
+{% load hq_shared_tags %}
+{% load menu_tags %}
+
+{% block content %}
+    <div>Unified Beneficiary</div>
+{% endblock content %}
+
+{% block js-inline %}
+    {{ block.super }}
+{% endblock %}
```

Comment:
Is this `block` necessary since it only calls super?

Reply:
I added this for future to remember that when we will be working on the unified beneficiary report UI we will only have to add the import of js file, something like example template for future reports